API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
+translation_of | ‘self’  
+
+
+
+<pre>Skills</pre>
+name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+image_url | UrlField  
+image_retina_url | UrlField  
+
+<pre>WebLitSkills</pre>
+name | CharField  
+short_name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+**Skills** | *:*  
+**Topic** | 1:*  
+
+<pre>Topic</pre>
+name | CharField  
+description | TextField  
+
+<pre>Competencies</pre>
+text | TextField  
+
+<pre>LessonPlan</pre>
+title | CharField  
+description | TextField  
+short_description | TextField  
+learning_objectives | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+level | CharField (choices)  
+developed_by | TextField  
```

Comment:
It's true that we've not been consistent so far (we haven't had a database behind these so far!) I feel like giving people the option to filter by author would be kind of cool/useful, but not necessarily a requirement. If it were, we would definitely need to be able to handle the multiple author use case and the no-link use case, though, since those are common.